docs/source/installation.rst

```diff
@@ -21,49 +21,159 @@ Requirements
 * `Requests <https://requests.readthedocs.io/>`_: used to get data from websites
 * `Pecos v0.1.9 <https://pecos.readthedocs.io/>`_: used for quality control analysis
 
-
 Install MHKiT-Python
 ^^^^^^^^^^^^^^^^^^^^^
 
-Option 1: PIP Install from Python
-""""""""""""""""""""""""""""""""""""
+Option 1: Using Conda
+"""""""""""""""""""""
 
-This option is recommended for MHKiT-Python users.
-To install MHKiT-Python using `pip <https://pip.pypa.io/en/stable/>`_::
+For most users, `Conda <https://docs.conda.io/projects/conda/en/stable/index.html>`_ is the preferred choice for installing MHKiT-Python. Conda includes the ``conda`` command line utility to manage Python versions and dependencies within `environments <https://business-docs.anaconda.com/en/latest/user/environment.html#:~:text=An%20environment%20is%20a%20folder,from%20other%20collections%20of%20packages.>`_. Anaconda provides a standardized Python build environment that is compatible across operating systems, simplifying MHKiT-Python installation and setup for all users.
 
-	pip install mhkit
 
-Using this option to install MHKiT-Python will automatically install Pecos.
+1. `Install Conda <https://docs.conda.io/projects/conda/en/stable/user-guide/install/index.html>`_ with Anaconda Distribution or Miniconda:
```

Comment:
reference the distribution comparison here https://docs.anaconda.com/distro-or-miniconda/